inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -33,12 +33,12 @@ params:
     value: "gamma"
     choices: ["beta", "binom", "cauchy", "chisq", "exp", "f", "gamma", "geom", "hyper", "lnorm", "logis", "nbinom", "norm", "pois", "smirnov", "t", "tukey", "unif", "weibull", "wilcox"]
   data_file: 
-    label: "Name of file containing the count data over time"
-    value: "data/covid_hosp_uk_20201024.xlsx"
+    label: "Name of file containing case data, whether a linelist or incidence data"
```

Comment:
I think line list should be two words.

Reply:
Fixed in [`e6f1bb3` (#137)](https://github.com/epiverse-trace/episoap/pull/137/commits/e6f1bb3e157498546fd806943f24691a1338bd14)